packages/datetime/src/dateRangeInput.tsx

```diff
@@ -36,10 +36,44 @@ import {
 } from "./dateRangePicker";
 
 export interface IDateRangeInputProps extends IDatePickerBaseProps, IProps {
+
+    /**
+     * The default date range to be used in the component when uncontrolled.
+     * This should not be set if `value` is set.
```

Comment:
Certainly. I copied this language from `dateInput.tsx`, so we should change there too.